We are building a specific git commit, which was pushed after the
release of 3.0.4, but before 4.0.0 (that isn't even available yet).

This should be removed after a future update.

--- chardet-master/chardet/version.py
+++ chardet-master/chardet/version.py
@@ -5,5 +5,5 @@ from within setup.py and from chardet su
 :author: Dan Blanchard (dan.blanchard@gmail.com)
 """
 
-__version__ = "4.0.0"
+__version__ = "3.0.4"
 VERSION = __version__.split('.')
